ctx.go

```diff
@@ -18,10 +18,10 @@ const (
 	maxBodySize = 1048576
 )
 
-// ctx is the context of the request.
+// Ctx is the context of the request.
 // It contains the request body, the path parameters, the query parameters, and the HTTP request.
 // Please do not use a pointer type as parameter.
-type ctx[B any] interface {
+type Ctx[B any] interface {
```

Comment:
You export it. I would prefer to see it exposed as Context and not Ctx

It would confuse people, including me.


```suggestion
type Context[B any] interface {
```